src/Uno.UI/UI/Xaml/Controls/ContentPresenter/ContentPresenter.cs

```diff
@@ -327,6 +327,27 @@ public FontStyle FontStyle
 		);
 	#endregion
 
+	#region FontStretch
+
+	public FontStretch FontStretch
+	{
+		get => (FontStretch)this.GetValue(FontStretchProperty);
+		set => this.SetValue(FontStretchProperty, value);
+	}
+
+	public static DependencyProperty FontStretchProperty { get; } =
+		DependencyProperty.Register(
+			nameof(FontStretch),
+			typeof(FontStretch),
+			typeof(ContentPresenter),
+			new FrameworkPropertyMetadata(
+				FontStretch.Normal,
+				FrameworkPropertyMetadataOptions.Inherits,
+				(s, e) => ((ContentPresenter)s)?.OnFontStretchChanged((FontStretch)e.OldValue, (FontStretch)e.NewValue)
+			)
+		);
```

Comment:
Let's use generated properties

Reply:
@jeromelaban For readability or perf?

My concern with generated DPs is that they cache the value when it changes. So, it's faster, but increases the object size. So my personal view to it is "speed vs memory"

Reply:
We can change later, if needed